src/applications/travel-pay/components/DocumentDownload.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { apiRequest } from '@department-of-veterans-affairs/platform-utilities/api';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+const DocumentDownload = ({
+  claimId,
+  documentId,
+  filename,
+  mimetype,
+  text,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  const downloadLinkHandler = async (e, docId) => {
+    setError('');
+    if (isLoading) {
+      return;
+    }
+    setIsLoading(true);
+    e.preventDefault();
+
+    try {
+      const response = await apiRequest(
+        `${
+          environment.API_URL
+        }/travel_pay/v0/claims/${claimId}/documents/${docId}`,
+      );
+
+      const byteCharacters = atob(response.data);
```

Comment:
Unless the frontend adds this somehow, there won't be a `data` field on the response from vets-api. I looked at [apiRequest](https://github.com/department-of-veterans-affairs/vets-website/blob/789a2b4190db019561b3be85037df623d76a0e3a/src/platform/utilities/api/index.js#L90), but couldn't see if the `data` attribute was being added by that code.

Reply:
Ah, okay thanks. This is one of the things I was unsure about.